packages/client/src/runtime/core/metrics/index.ts

```diff
@@ -0,0 +1 @@
+export { MetricsClient } from './MetricsClient'
```

Comment:
Is this file/pattern in case we want to have better/centralized exports?

Reply:
So, here I want two things:

1. Centralized place for importing anything metrics-related, hence separate folder with `index.ts`
2. Still being to import to open `MetricsClient.ts` file by name, hence `index.ts` doing just a re-export

Admittedly, "anything metrics-related" right now is only `MetricsClient` class and it will quite likely stay that way, so I guess we could remove this file. What would you prefer?